dracut/30ignition/boot.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Ignition Boot Mount
+
```

Comment:
Maybe a `Documentation=https://github.com/coreos/ignition` like the other ones here?

Reply:
Added.